Xplat/src/main/java/vazkii/botania/common/entity/ManaSparkEntity.java

```diff
@@ -59,7 +59,11 @@ public class ManaSparkEntity extends SparkBaseEntity implements ManaSpark {
 
 	private final Set<ManaSpark> transfers = Collections.newSetFromMap(new WeakHashMap<>());
```

Comment:
not strictly related to the pr, but I think renaming this to `outgoingTransfers` might make the code clearer